client/app/visualizations/pivot/index.js

```diff
@@ -41,6 +42,22 @@ function pivotTableRenderer() {
             // We need to give the pivot table its own copy of the data, because it changes
             // it which interferes with other visualizations.
             data = angular.copy($scope.queryResult.getData());
+
+            // override moment.fn.toString for formatting.
+            const dateTimeFormat = {
+              date: clientConfig.dateFormat || 'DD/MM/YY',
+              datetime: clientConfig.dateTimeFormat || 'DD/MM/YY HH:mm',
+            };  
+            each($scope.queryResult.getColumns(), (column) => {
+              if (['date', 'datetime'].indexOf(column.type) >= 0) {
+                each(data, (datum) => {
+                  datum[column.name].toString = function f() {
+                    return this.format(dateTimeFormat[column.type]);
```

Comment:
Changing this line to `return this.local().format(dateTimeFormat[column.type]);` to make it local will fix timezone issue in pivot viz, but then it will contradict values shown in Table viz.

@kravets-levko, WDYT?